javascript/cra-starter/README.md

```diff
@@ -1 +1,44 @@
-# Basic CRA Starter
```

Comment:
now that we have this on the example repo we should get rid of this one as a whole, also we should add a link to all of the template repos in the README of this one